pkg/pillar/agentlog/agentlog.go

```diff
@@ -444,27 +464,33 @@ func spoofStdFDs(agentName string) *os.File {
 }
 
 func Init(agentName string) {
-	savedAgentName = agentName
-	savedPid = os.Getpid()
-	log.SetOutput(os.Stdout)
-	originalStdout := spoofStdFDs(agentName)
-	log.SetOutput(originalStdout)
-	hook := new(FatalHook)
-	log.AddHook(hook)
-	hook2 := new(SourceHook)
-	log.AddHook(hook2)
-	// Report nano timestamps
-	formatter := log.JSONFormatter{
-		TimestampFormat: time.RFC3339Nano,
-	}
-	log.SetFormatter(&formatter)
-	log.SetReportCaller(true)
-	log.RegisterExitHandler(printStack)
-
-	sigs := make(chan os.Signal, 1)
-	signal.Notify(sigs, syscall.SIGUSR1)
-	signal.Notify(sigs, syscall.SIGUSR2)
-	go handleSignals(sigs)
+	agentPid := os.Getpid()
```

Comment:
Wait, @eriknordmark stdout is still being redirected here -- I'm confused wrt. subject of this PR then

Reply:
Avoid doing it twice if Init is called twice. This was the top issue for @adarsh-zededa

Reply:
Great -- that's what I thought looking at the PR @eriknordmark -- just couldn't reconcile it with the PR's original title